bigfun/cli.py

```diff
@@ -101,32 +100,35 @@ def generate_bigfunctions_list_markdown(bigfunctions):
     help_headers_color='yellow',
     help_options_color='cyan'
 )
-def cli():
-    pass
+@click.pass_context
```

Comment:
I understand ctx is an advanced click functionality which enables, for instance, to factor common arguments management. But it seems you don't make use of it. So I find it adds complexity while not bringing any value.